packages/storybook/stories/CallComposite/snippets/Container.snippet.tsx

```diff
@@ -23,15 +24,15 @@ export type ContainerProps = {
 };
 
 const isTeamsMeetingLink = (link: string): boolean => link.startsWith('https://teams.microsoft.com/l/meetup-join');
-const isGroupLink = (link: string): boolean => link.indexOf('-') !== -1;
+const isGroupID = (id: string): boolean => uuid.validate(id);
 
-const createLocator = (link: string): CallAdapterLocator => {
-  if (isTeamsMeetingLink(link)) {
-    return { meetingLink: link };
-  } else if (isGroupLink(link)) {
-    return { groupId: link };
+const createCallAdapterLocator = (locator: string): CallAdapterLocator => {
+  if (isTeamsMeetingLink(locator)) {
+    return { meetingLink: locator };
+  } else if (isGroupID(locator)) {
+    return { groupId: locator };
   }
-  return { roomId: link };
+  return { roomId: locator };
```

Comment:
(non-blocking comment)
is there a way to test if it is a roomId?
Would be nice to do:
```
if (isTeamsLink()) return { meetingLink: loc }
else if (isGroupId()) return { groupId: loc }
else if (isRoomsId()) return { roomId: loc }
else {
  throw new Error(`Unrecognized locator: ${locator}`);
}
```

Reply:
To ensure if we start supporting a new locator this can't assume its a roomsId